docs/public/docs/pipeline-example-3.md

```diff
@@ -1,37 +0,0 @@
-# Reading and writing from/to ClickHouse
```

Comment:
An orphaned file - not accessible from the menu, remnant of an old documentation, info here is not up-to-date. The only way to access this file was to either know its URL or via search.